techmatters/flex-plugins
# Utilize label in addParticipant api for 4 way conference

Primary reviewer:

## Description
- Use label for using `addParticipant` serverless api call in order to add more than 3 participants to conference
- The participant `label` sent to the api is denoted as `Participant x` where `x` represents the number as per sequence assigned to each participant upon joining.

### Checklist

### Related Issues
Fixes # https://tech-matters.slack.com/archives/CV4QM23UJ/p1686681984027449?thread_ts=1686681980.315439&cid=CV4QM23UJ

### Verification steps
- Start a conference call and add participants until there are 4 in total. This should work now with no errors.
